Description.md

```diff
@@ -522,6 +522,25 @@ MindTheWord एक माध्यमिक भाषा सीखने के
 
 अपनी भाषा सीखने आप खुशी और अबाध भाषण ला सकते हैं !
 
+Gujarati
```

Comment:
Is there an international language code for Gujarati? (For example, the code for Ukrainian (below) is "ua", the code for english is "en", and so on...)